docker-compose.yml

```diff
@@ -46,7 +46,7 @@ services:
     << : *default-volumes
     environment:
       << : *default-environment
-      LAGOON_LOCALDEV_URL: ${LOCALDEV_URL_NGINX:-http://govcms-nginx.docker.amazee.io}
+      LAGOON_LOCALDEV_URL: ${LOCALDEV_URL:-http://govcms.docker.amazee.io}
```

Comment:
@thom8 not sure about this now. There are 2 variable for this. which one to use?

Reply:
pls use `LOCALDEV_URL` and drop `LOCALDEV_URL_NGINX` also in `.env.default`